app/reports/spree/cart_additions_report.rb

```diff
@@ -19,9 +19,8 @@ def sku
 
     def report_query
       Spree::CartEvent
-        .added
-        .joins(:variant)
-        .joins(:product)
+        .where(activity: 'add')
```

Comment:
@harsh-ag  Can you please move this to a scope with 'add' being sent as a parameter